Makefile

```diff
@@ -102,3 +110,13 @@ ci-port-forward:
 
 ci-dispatch:
 	curl -v -H "content-type: application/json" -H "Authorization: PSK ${PSK}" -d "@examples/payload.json" http://localhost:8000/internal/dispatch
+
+run-lint:
```

Comment:
instead of inlining the two tasks together you can still define them separately but use Makefile dependencies to tie them together, so that the install tasks is run automatically before each run task